### ISSUE TITLE
[Feg] Cleanup references to XWF

### ISSUE THREAD

**Issue by ssanadhya** [magma/magma#13546]
Once https://github.com/magma/magma/pull/13545 is merged, the XWF code will be removed.
- [x] remove scripts (PR: https://github.com/magma/magma/pull/13592)
- [x] remove frontend xwf implementation (PR: https://github.com/magma/magma/pull/13603)
- [x] remove option to configure xwf from swagger/backend (https://github.com/magma/magma/pull/13627)
- [x] remove the pass through controller in pipelined (PR: https://github.com/magma/magma/pull/13621)
- [x] remove xwf references from helm charts (PR: https://github.com/magma/magma/pull/13619)
- [x] after all PRs have been merged, check for remaining xwf occurrances again (orc8r/cloud/api/v1/ is not considered here, since it is not kept up to date. Regenerating leads to a massive set of changes, 1000+ files.)
   https://github.com/magma/magma/pull/13668
   https://github.com/magma/magma/pull/13667
   https://github.com/magma/magma/pull/13677
- [x] prepare sprint review
- [x] open follow up ticket for radius (https://github.com/magma/magma/issues/13671)
- [x] remove xwf approvers team from Github

Go radius is out of scope of this ticket.

This issue is to track clean-up for XWF references from other components of Magma core, e.g. Federation Gateway (feg)

**Comment by crasu** [comment]
There are only a few references left
```
rg --files-with-matches xwf
CODEOWNERS
bazel/scripts/check_py_bazel.sh
nms/generated/api.ts
orc8r/tools/docker/install_gateway.sh
nms/server/network/routes.ts
nms/server/grafana/handlers.ts
nms/shared/types/network.ts
feg/radius/src/deploy.sh
lte/gateway/python/magma/pipelined/BUILD.bazel
lte/gateway/python/magma/pipelined/service_manager.py
feg/radius/src/server/sse_listener_test.go
feg/radius/src/loader/static_loader.go
feg/radius/src/monitoring/init.go
nms/app/components/layout/__tests__/useSections-test.tsx
lte/gateway/python/magma/pipelined/app/xwf_passthru.py
lte/gateway/python/magma/pipelined/app/BUILD.bazel
orc8r/cloud/api/v1/go/models/network_carrier_wifi_configs.go
feg/radius/lib/go/libgraphql/xwf_customer.go
feg/radius/src/monitoring/scuba/scuba_test.go
feg/radius/src/monitoring/scuba/scuba.go
feg/radius/src/modules/xwfv3/xwfv3.go
feg/radius/src/modules/adaptruckus/adapt_ruckus_test.go
feg/radius/src/config/samples/radius.xwfv3.config.json.template
orc8r/cloud/go/services/obsidian/swagger/v1/swagger.yml
feg/radius/lib/go/radius/coa/coa_sender.go
feg/radius/lib/go/radius/coa/coa_sender_test.go
feg/gateway/helm/feg-kubevirt/templates/bin/_install_gateway.sh.tpl
feg/gateway/helm/feg-virtlet/templates/bin/_install_gateway.sh.tpl
cwf/gateway/helm/cwf-kubevirt/templates/bin/_install_gateway.sh.tpl
cwf/gateway/helm/cwf-virtlet/templates/bin/_install_gateway.sh.tpl
feg/radius/src/modules/xwfv3/xwfhttp2/http2client.go
feg/radius/src/modules/xwfv3/xwfhttp2/http2client_test.go
feg/radius/lib/go/radius/dictionaries/xwf/generated.go
feg/radius/lib/go/radius/dictionaries/xwfradiusserver/generated.go
cwf/cloud/go/services/cwf/obsidian/models/network_carrier_wifi_configs_swaggergen.go
cwf/cloud/go/services/cwf/obsidian/models/swagger.v1.yml
Line count: 36
```
